src/main.cpp

```diff
@@ -74,6 +81,54 @@ int main(int argc, char *argv[]){
             return 2;
         }
     }
+
+    // Generate the seed values:
+    char seq[100];
+    std::ifstream file("/dev/urandom", std::ios::in | std::ios::binary);
+    if(!file) {
+        std::cout << "Error opening file\n";
+    }
+    file.read(seq, sizeof(char)*100);
+    file.close();
+    
+    std::seed_seq seed(seq, seq+100);
+
+    std::mt19937_64 gen(seed);
+
+    std::uniform_int_distribution<int64_t> dist(0, 100000);
+    
+    // Set of Characters
+    std::string CharSet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+    // Set of symbol characters
+    std::string symblSet = "!@#$%^&*-_";
+
     
+    if(!symbol_set && !askd_VorH){
+        // Instansiate the RadomEngine class: for various cofigurations.
+        // 1. Instansiate the class RandomEngine for -L option.
+        RandomEngine el(length);
+        el.setSymbolSet(symbol_set);
```

Comment:
No point in settings members with helper functions even when we have constructors to do that.